src/FlameGraph.js

```diff
@@ -29,12 +31,16 @@ export default class FlameGraph extends PureComponent<Props, State> {
   // Memoize this wrapper object to avoid breaking PureComponent's sCU.
   // Attach the memoized function to the instance,
   // So that multiple instances will maintain their own memoized cache.
-  getItemData = memoize((data, focusedNode, focusNode, width) => ({
-    data,
-    focusedNode,
-    focusNode,
-    scale: value => value / focusedNode.width * width,
-  }));
+  getItemData = memoize(
+    (data, focusedNode, focusNode, onMouseOut, onMouseOver, width) => ({
+      data,
+      focusedNode,
+      focusNode,
+      scale: value => value / focusedNode.width * width,
+      onMouseOut,
+      onMouseOver,
```

Comment:
People often recreate event handlers like `onMouseOut`/`onMouseOver` inline, which would effectively break this memoization. We might need to emphasize in the docs that in this case, using a callback hook would be appropriate (even though it's not necessary for native event handlers).

I wonder if a different name (that doesn't sound the same as the native mouse handlers) might help avoid some potential confusion?